src/tfrobot/tfrobot_config.md

```diff
@@ -0,0 +1,119 @@
+<h1> Configuration File</h1>
+
+<h2>Table of Contents</h2>
+
+- [Introduction](#introduction)
+- [Examples](#examples)
+  - [YAML Example](#yaml-example)
+  - [JSON Example](#json-example)
+- [Create a Configuration File](#create-a-configuration-file)
+
+***
+
+## Introduction
+
+To use TFROBOT, the user needs to create a YAML or a JSON configuration file that will contain the mass deployment information, such as the groups information, number of VMs to deploy how, the compute, storage and network resources needed, as well as the user's credentials, such as the SSH public key, the network (main, test, dev, qa) and the TFChain mnemonics.
+
+## Examples
+
+We present here a configuration file example that deploys 3 nodes with 2 vcores, 16GB of RAM, 100GB of SSD, 50GB of HDD and an IPv4 address. The same deployment is shown with a YAML file and with a JSON file. Parsing is based on file extension, TFROBOT will use JSON format if the file has a JSON extension and YAML format otherwise.
+
+You can use this example for guidance, and make sure to replace placeholders and adapt the groups based on your actual project details. To the minimum, `ssh_key1` should be replaced by the user SSH public key and `example-mnemonic` should be replaced by the user mnemonics.
+
+### YAML Example
+
+```
+node_groups:
+  - name: group_a
+    nodes_count: 3
+    free_cpu: 2
+    free_mru: 16
+    free_ssd: 100
+    free_hdd: 50
+    pubip4: true
+    pubip6: false
```

Comment:
- You have added the public IPs in the node group (which works as a node filter), but in the VMs group you didn't add them, so you will filter for nodes with public IPs only to deploy a VM with no IPs.
- And can you also add a note if it isn't here that if no IPs are specified as true, the bot will automatically add ygg ip to the VM (at least one IP should be true to be able to ssh to the VM).
- Its `public_ip4` not pubip4, and `public_ip6` not pubip6.

Reply:
Done. Took the opportunity to add dedicated, certified and region. Let me know if things are off!

Thanks.